tests/queryset_pickle/models.py

```diff
@@ -60,3 +60,12 @@ class SomeModel(models.Model):
 
 class M2MModel(models.Model):
     groups = models.ManyToManyField(Group)
+
+
+class Parent(models.Model):
```

Comment:
Did try using the `Group` / `Event` models? Maybe the custom managers on `Group` were a problem.